Library/Notifier.swift

```diff
@@ -14,35 +13,19 @@ struct Notifier {
     private static func scheduleNotification(for location: Location, at date: Date) {
         guard let location = Location.current else { return }
 
-        let dateFormatter = DateFormatter()
-        dateFormatter.dateStyle = DateFormatter.Style.long
-
-        let notificationID = dateFormatter.string(from: date)
+        let notificationID = String(DateHasher.hashValue(for: date))
         let sunriseDate = location.sunTime.sunriseStartTime(for: date)
 
         let formattedMessage = self.formattedMessage(location: location, date: date)
 
-        // TODO: Fix deprecated UILocalNotification
         UILocalNotification.schedule(notificationID, at: sunriseDate, message: formattedMessage)
     }
 
     static func formattedMessage(location: Location, date: Date) -> String {
-        let interval = location.sunTime.daylightLengthDifference(on: date)
-
-        // If the time differential is smaller than a minute, say 1 minute instead.
-        let minutesRounded = max(abs(Int(Darwin.round(interval / 60.0))), 1)
-
-        let generatedMessage = Message.notificationMessage(for: date, coordinates: location.coordinates)
-
-        let format = NSLocalizedString("number_of_minutes", comment: "")
-        let minuteString = String.localizedStringWithFormat(format, minutesRounded)
-        let formattedMessage = String(format: generatedMessage, minuteString)
-
-        return formattedMessage
+        return Message.notificationMessage(for: date, coordinates: location.coordinates)
     }
 
     static func cancelAllNotifications() {
-        // TODO: Fix deprecated UILocalNotification
```

Comment:
any particular reason you removed the `TODO`s on these?

Reply:
Kind of a double-warning, since we already have the warning for using deprecated APIs. 👍

Reply:
Why aren't we actually fixing the warning?

Reply:
That would be my next step. 👍